app/controllers/ems_common.rb

```diff
@@ -258,6 +258,7 @@ def show_list
   end
 
   def new
+    @doc_url = self.documentation_url if respond_to? :documentation_url
```

Comment:
This should probably happen in `edit` as well as `new`. And possibly `create`, though I'm not sure about that one..